scipy/stats/_hypotests.py

```diff
@@ -449,15 +405,15 @@ def _tau_b(A):
         return np.nan, np.nan
 
     NA = A.sum()
-    PA = _P(A)
-    QA = _Q(A)
+    PA = _P_pythran(A)
```

Comment:
Would it be better to not rename this? The code could remain unchanged. Same for other variables.